eng/scripts/InstallationCheck.ps1

```diff
@@ -9,24 +9,33 @@ param (
     [string] $PipelineWorkspace
```

Comment:
It is probably best to combine these 3 into a single parameter with the full path to the artifact directory.

Reply:
Well I guess we might also need the Artifact or Package name as well, but `$PipelineWorkspace/$ArtifactsDirectory-signed` should just be one directory as we don't want to hard-code the "-signed" name inside of our scripts it should remain in the yml file.